doc/BENCHMARKING.md

```diff
@@ -22,12 +22,11 @@ We keep the records of benchmarking results in `gh-pages` branch and the perform
 While our benchmark script is written for CI testing only, it is possible to run manually.
 You can use the following steps.
 
-1. Install dependencies by running below commands for Ubuntu.
+1. Install dependencies and configure FUSE by running the following script in the repository:
```

Comment:
apt-get does not work outside Linux, right?

Reply:
It won't work outside of Debian-based operating systems (so Ubuntu, Debian).

Reply:
Also, the script only works on distros using apt/apt-get and rpm/yum/dnf.